forms-flow-web/src/components/ServiceFlow/details/TaskHeader.js

```diff
@@ -140,16 +112,7 @@ const TaskHeader = React.memo(() => {
     dispatch(
       // eslint-disable-next-line no-unused-vars
       updateBPMTask(taskId, updatedTask, (err, response) => {
```

Comment:
and you can optimize these function calls like this `disatch(updateBPMTask(taskId, updatedTask,updateBpmTaskDetails))` no need to write a function for it again. it will pass the err and response directly into the function  this is an example and follow these methods above function calls also

Reply:
updated